flask_core/helpers.py

```diff
@@ -22,3 +23,25 @@ def get_database_type(uri):
         type = "postgres"
 
     return type
+
+
+def log_request():
+    from flask import g, current_app, request
+
+    log_line = (
+        textwrap.dedent(
+            """
+        [REQUEST] {}
+    """
+        )
+        .format(
+            json.dumps(
+                dict(
+                    zid=getattr(g, "zid", "unknown user"), method=request.method, uri=request.url, payload=request.form
```

Comment:
jesus this is good style according to our linter?

Reply:
Fixed the style, it was 2 am shut up